OpenRA.Mods.Common/Scripting/Global/MapGlobal.cs

```diff
@@ -110,13 +113,25 @@ public LuaValue LobbyOption(string id)
 
 			if (option == null)
 			{
-				Log.Write("lua", "A ScriptLobbyDropdown with ID `" + id + "` was not found.");
+				Log.Write("lua", $"A {nameof(ScriptLobbyDropdown)} with ID `{id}` was not found.");
 				return null;
 			}
 
 			return option.Value;
 		}
 
+		[Desc("Returns the value of a `ScriptLobbyDropdown` selected in the game lobby or fallback to a default value.")]
+		public LuaValue LobbyOptionOrDefault(string id, string backfall)
```

Comment:
```suggestion
		public LuaValue LobbyOptionOrDefault(string id, string fallback)
```
Or `default` (which might fit better considering the method name and description).

Reply:
`default` is a C# keyword.